src/tasks/level10.ts

```diff
@@ -97,7 +99,8 @@ export const GiantQuest: Quest = {
       completed: () => have($item`amulet of extreme plot significance`),
       do: $location`The Penultimate Fantasy Airship`,
       choices: () => {
-        return { 178: 2, 182: have($item`model airship`) ? 1 : 4, 1387: 3 };
```

Comment:
I'm confused on this change, is this still of equal value or an improvement for those without batwings? Dunno if the choice changed.